tests/models/utils.py

```diff
@@ -182,7 +179,7 @@ def load_model_from_checkpoint(root_weights_dir, module_class=LightningTemplateM
     return trained_model
 
 
-def run_prediction(dataloader, trained_model, dp=False, min_acc=0.45):
+def run_prediction(dataloader, trained_model, dp=False, min_acc=0.35):
```

Comment:
did you lower the acc because 45 was not anymore achievable? is it really necessary to go that low?

Reply:
randomly like 5% of test got 0.37 so in future with custom MNIST I would use just 5 digits and then increase this acc back... the point is that the small MNIST of 2k samples we have about 200 samples per class which is not much...